TokenManager.js

```diff
@@ -338,14 +344,16 @@
             }
         },
         serializeResponse: function (response) {
-            return response.text().then((bodyText) => ({
-                body: bodyText,
-                init: {
-                    status: response.status,
-                    statusText: response.statusText,
-                    headers: this.serializeHeaders(response.headers)
-                }
-            }));
+            return response.text().then((bodyText) => {
+                return {
+                    body: bodyText,
+                    init: {
+                        status: response.status,
+                        statusText: response.statusText,
+                        headers: this.serializeHeaders(response.headers)
+                    }
+                };
+            });
```

Comment:
I'm detecting a trend - you don't like implicit returns from a fat arrow function? :)